runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -208,14 +208,16 @@ private void sendLinuxNotification(
 
 		executorService.submit(() ->
 		{
-			final boolean success = sendCommand(commands)
-					.map(process -> process.exitValue() == 0)
-					.orElse(false);
-
-			if (!success)
+			try
 			{
-				sendTrayNotification(title, message, type);
+				sendCommand(commands, 500);
```

Comment:
This is wrong because it will always fall back to the tray notif. regardless of it the command succeeds.